<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />

<meta name="viewport" content="width=device-width, initial-scale=1">

<meta name="author" content="Timothy A. Myers, Stephen J. Chanock and Mitchell J. Machiela" />

<meta name="date" content="2019-07-30" />

<title>LDlinkR: an R package for rapidly calculating linkage disequilibrium statistics</title>



<style type="text/css">code{white-space: pre;}</style>
<style type="text/css" data-origin="pandoc">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */

</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    for (var j = 0; j < rules.length; j++) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") continue;
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' || rule.style.backgroundColor === '') continue;
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>



<style type="text/css">body {
background-color: #fff;
margin: 1em auto;
max-width: 700px;
overflow: visible;
padding-left: 2em;
padding-right: 2em;
font-family: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
font-size: 14px;
line-height: 1.35;
}
#header {
text-align: center;
}
#TOC {
clear: both;
margin: 0 0 10px 10px;
padding: 4px;
width: 400px;
border: 1px solid #CCCCCC;
border-radius: 5px;
background-color: #f6f6f6;
font-size: 13px;
line-height: 1.3;
}
#TOC .toctitle {
font-weight: bold;
font-size: 15px;
margin-left: 5px;
}
#TOC ul {
padding-left: 40px;
margin-left: -1.5em;
margin-top: 5px;
margin-bottom: 5px;
}
#TOC ul ul {
margin-left: -2em;
}
#TOC li {
line-height: 16px;
}
table {
margin: 1em auto;
border-width: 1px;
border-color: #DDDDDD;
border-style: outset;
border-collapse: collapse;
}
table th {
border-width: 2px;
padding: 5px;
border-style: inset;
}
table td {
border-width: 1px;
border-style: inset;
line-height: 18px;
padding: 5px 5px;
}
table, table th, table td {
border-left-style: none;
border-right-style: none;
}
table thead, table tr.even {
background-color: #f7f7f7;
}
p {
margin: 0.5em 0;
}
blockquote {
background-color: #f6f6f6;
padding: 0.25em 0.75em;
}
hr {
border-style: solid;
border: none;
border-top: 1px solid #777;
margin: 28px 0;
}
dl {
margin-left: 0;
}
dl dd {
margin-bottom: 13px;
margin-left: 13px;
}
dl dt {
font-weight: bold;
}
ul {
margin-top: 0;
}
ul li {
list-style: circle outside;
}
ul ul {
margin-bottom: 0;
}
pre, code {
background-color: #f7f7f7;
border-radius: 3px;
color: #333;
white-space: pre-wrap; 
}
pre {
border-radius: 3px;
margin: 5px 0px 10px 0px;
padding: 10px;
}
pre:not([class]) {
background-color: #f7f7f7;
}
code {
font-family: Consolas, Monaco, 'Courier New', monospace;
font-size: 85%;
}
p > code, li > code {
padding: 2px 0px;
}
div.figure {
text-align: center;
}
img {
background-color: #FFFFFF;
padding: 2px;
border: 1px solid #DDDDDD;
border-radius: 3px;
border: 1px solid #CCCCCC;
margin: 0 5px;
}
h1 {
margin-top: 0;
font-size: 35px;
line-height: 40px;
}
h2 {
border-bottom: 4px solid #f7f7f7;
padding-top: 10px;
padding-bottom: 2px;
font-size: 145%;
}
h3 {
border-bottom: 2px solid #f7f7f7;
padding-top: 10px;
font-size: 120%;
}
h4 {
border-bottom: 1px solid #f7f7f7;
margin-left: 8px;
font-size: 105%;
}
h5, h6 {
border-bottom: 1px solid #ccc;
font-size: 105%;
}
a {
color: #0033dd;
text-decoration: none;
}
a:hover {
color: #6666ff; }
a:visited {
color: #800080; }
a:visited:hover {
color: #BB00BB; }
a[href^="http:"] {
text-decoration: underline; }
a[href^="https:"] {
text-decoration: underline; }

code > span.kw { color: #555; font-weight: bold; } 
code > span.dt { color: #902000; } 
code > span.dv { color: #40a070; } 
code > span.bn { color: #d14; } 
code > span.fl { color: #d14; } 
code > span.ch { color: #d14; } 
code > span.st { color: #d14; } 
code > span.co { color: #888888; font-style: italic; } 
code > span.ot { color: #007020; } 
code > span.al { color: #ff0000; font-weight: bold; } 
code > span.fu { color: #900; font-weight: bold; }  code > span.er { color: #a61717; background-color: #e3d2d2; } 
</style>

</head>

<body>




<h1 class="title toc-ignore">LDlinkR: an R package for rapidly calculating linkage disequilibrium statistics</h1>
<h4 class="author"><em>Timothy A. Myers, Stephen J. Chanock and Mitchell J. Machiela</em></h4>
<h4 class="date"><em>30 Jul 2019</em></h4>



<hr />
<div id="description" class="section level2">
<h2>Description</h2>
<p>LDlink is an interactive and powerful suite of web-based tools for querying germline variants in population groups of interest to generate interactive tables and plots.</p>
<p><em>LDlinkR</em> is an R package developed to query and download results generated by LDlink web-based applications from the R console. <em>LDlinkR</em> accelerates genomic research by providing efficient and user-friendly functions to programmatically interrogate pairwise linkage disequilibrium from large lists of variants.</p>
</div>
<div id="installation" class="section level2">
<h2>Installation</h2>
<ul>
<li>The stable version of <em>LDlinkR</em> can be installed from CRAN:</li>
</ul>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1"></a><span class="kw">install.packages</span>(<span class="st">&quot;LDlinkR&quot;</span>)</span></code></pre></div>
<ul>
<li>The development version of the <em>LDlinkR</em> package can be installed from the GitHub repository by using the <em>devtools</em> package:</li>
</ul>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1"></a>devtools<span class="op">::</span><span class="kw">install_github</span>(<span class="st">&quot;CBIIT/LDlinkR&quot;</span>)</span></code></pre></div>
<p><em>LDlinkR</em> depends on the following package:</p>
<ul>
<li><em>utils</em>, version 3.5.2 or later</li>
<li><em>httr</em>, version 1.4.0 or later</li>
</ul>
</div>
<div id="personal-access-token---required" class="section level2">
<h2>Personal Access Token - Required</h2>
<p>In order to access the LDlink API via <em>LDlinkR</em>, we use a personal access token. This is a common convention followed by many APIs and emulates the more familiar HTTPS username/password or SSH keys.</p>
<p>You will need to:</p>
<ul>
<li>Make a one-time request for your personal access token from a web browser at <a href="https://ldlink.nci.nih.gov/?tab=apiaccess" class="uri">https://ldlink.nci.nih.gov/?tab=apiaccess</a>.</li>
<li>Once registered, your personal access token will be emailed to you. It is a string of 12 random letters and numbers.</li>
<li>Provide it as an argument when using <em>LDlinkR</em> (See examples below).</li>
</ul>
<p>(Optional) The best security practice is to store your personal access token as an envionment variable where <em>LDlinkR</em> can find it and use it on your behalf but where it will not be accidentally shared with the public. Step-by-step instructions follow:</p>
<p>After retrieving your personal access token from your email, put your token in your <code>.Renviron</code> file. <code>.Renviron</code> is a hidden file that lives in your home directory. The easiest way to both find and edit the <code>.Renviron</code> file is with a function from the <em>usethis</em> package. From the R console, do:</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1"></a>usethis<span class="op">::</span><span class="kw">edit_r_environ</span>()</span></code></pre></div>
<p>Your <code>.Renviron</code> file should open in your editor. Add a line that looks like this:</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1"></a>LDLINKR_TOKEN=yourtokenhere123</span></code></pre></div>
<p><strong>Important</strong>, ensure you put a line break at the end by hitting the <em>enter/return</em> key.</p>
<p>Save and close the <code>.Renviron</code> file. Restart R, as environment variables are only loaded from <code>.Renviron</code> at the start of a new R session. Now, check to see that your token is available by entering:</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1"></a><span class="kw">Sys.getenv</span>(<span class="st">&quot;LDLINKR_TOKEN&quot;</span>)</span></code></pre></div>
<pre><code>## [1] &quot;yourtokenhere123&quot;</code></pre>
<p>You should see your personal access token print to the screen, as shown above. Now, <em>LDlinkR</em> function calls that use</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1"></a><span class="kw">Sys.getenv</span>(<span class="st">&quot;LDLINKR_TOKEN&quot;</span>)</span></code></pre></div>
<p>for the <code>token</code> argment in <em>LDlinkR</em> function calls will use your personal access token in a private and secure way. This method will be used in the extended examples that follow.</p>
<hr />
</div>
<div id="functions" class="section level2">
<h2>Functions</h2>
<div id="ldhap" class="section level3">
<h3>LDhap</h3>
<div id="function" class="section level4">
<h4>Function</h4>
<pre><code>LDhap(snps, pop=&quot;CEU&quot;, token=NULL, file = FALSE)</code></pre>
<p>Calculates population specific haplotype frequencies of all haplotypes observed for a list of query variants. Input is a list of variant RS numbers (concatenated list) and a population group.</p>
</div>
<div id="arguments" class="section level4">
<h4>Arguments</h4>
<ul>
<li><code>snps</code>, a list of between 1 - 30 variants, using an rsID or chromosome coordinate (e.g. “chr7:24966446”)</li>
<li><code>pop</code>, a 1000 Genomes Project population, uses three letter population code, (e.g. YRI or CEU), multiple allowed, default = “CEU”</li>
<li><code>token</code>, LDlink provided user access token is required, default = NULL</li>
<li><code>file</code>, optional character string naming a path and file for saving results. If file = FALSE, no file will be generated, default = FALSE</li>
</ul>
</div>
<div id="usage-multiple-query-variants-single-population" class="section level4">
<h4>Usage: Multiple query variants, single population</h4>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1"></a><span class="kw">LDhap</span>(<span class="dt">snps =</span> <span class="kw">c</span>(<span class="st">&quot;rs3&quot;</span>, <span class="st">&quot;rs4&quot;</span>, <span class="st">&quot;rs148890987&quot;</span>), </span>
<span id="cb9-2"><a href="#cb9-2"></a>      <span class="dt">pop =</span> <span class="st">&quot;CEU&quot;</span>, </span>
<span id="cb9-3"><a href="#cb9-3"></a>      <span class="dt">token =</span> <span class="kw">Sys.getenv</span>(<span class="st">&quot;LDLINK_TOKEN&quot;</span>)</span>
<span id="cb9-4"><a href="#cb9-4"></a>     )</span></code></pre></div>
<pre><code>##   rs148890987 rs3 rs4 Count Frequency
## 1           C   C   A   176    0.8889
## 2           T   T   G    11    0.0556
## 3           T   C   A     7    0.0354
## 4           C   T   G     4    0.0202</code></pre>
<p><br></p>
</div>
<div id="usage-multiple-query-variants-multiple-populations" class="section level4">
<h4>Usage: Multiple query variants, multiple populations</h4>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1"></a><span class="kw">LDhap</span>(<span class="dt">snps =</span> <span class="kw">c</span>(<span class="st">&quot;rs3&quot;</span>, <span class="st">&quot;rs4&quot;</span>, <span class="st">&quot;rs148890987&quot;</span>),</span>
<span id="cb11-2"><a href="#cb11-2"></a>      <span class="dt">pop =</span> <span class="kw">c</span>(<span class="st">&quot;YRI&quot;</span>, <span class="st">&quot;CEU&quot;</span>),</span>
<span id="cb11-3"><a href="#cb11-3"></a>      <span class="dt">token =</span> <span class="kw">Sys.getenv</span>(<span class="st">&quot;LDLINK_TOKEN&quot;</span>)</span>
<span id="cb11-4"><a href="#cb11-4"></a>     )</span></code></pre></div>
<pre><code>##   rs148890987 rs3 rs4 Count Frequency
## 1           C   C   A   355    0.8575
## 2           C   T   G    41     0.099
## 3           T   T   G    11    0.0266
## 4           T   C   A     7    0.0169</code></pre>
<p>Output is a table of alleles, haplotype count and haplotype frequencies.</p>
<hr />
</div>
</div>
<div id="ldmatrix" class="section level3">
<h3>LDmatrix</h3>
<div id="function-1" class="section level4">
<h4>Function</h4>
<pre><code>LDmatrix(snps, pop = &quot;CEU&quot;, r2d = &quot;r2&quot;, token = NULL, file = FALSE)</code></pre>
<p>Generates a data frame of pairwise linkage disequilibirum statistics. Input is a list of between 2 to 1000 variants. Desired output can be based on estimates of R<sup>2</sup> or D’.</p>
</div>
<div id="arguments-1" class="section level4">
<h4>Arguments</h4>
<ul>
<li><code>snps</code>, list of between 2 - 1,000 variants, using an rsID or chromosome coordinate (GRCh37/hg19) (e.g. “chr7:24966446”)</li>
<li><code>pop</code>, a 1000 Genomes Project population, uses three letter population code, (e.g. YRI or CEU), multiple allowed, default = “CEU”</li>
<li><code>r2d</code>, use either “r2” for pairwise R<sup>2</sup> statistics or “d” for pairwise D’ statistics</li>
<li><code>token</code>, LDlink provided user access token is required, default = NULL</li>
<li><code>file</code>, optional character string naming a path and file for saving results. If file = FALSE, no file will be generated, default = FALSE</li>
</ul>
</div>
<div id="usage-multiple-query-variants-single-population-r2" class="section level4">
<h4>Usage: Multiple query variants, single population, R<sup>2</sup></h4>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1"></a><span class="kw">LDmatrix</span>(<span class="dt">snps =</span> <span class="kw">c</span>(<span class="st">&quot;rs496202&quot;</span>, <span class="st">&quot;rs11147477&quot;</span>, <span class="st">&quot;rs201578600&quot;</span>), </span>
<span id="cb14-2"><a href="#cb14-2"></a>         <span class="dt">pop =</span> <span class="st">&quot;YRI&quot;</span>, <span class="dt">r2d =</span> <span class="st">&quot;r2&quot;</span>, </span>
<span id="cb14-3"><a href="#cb14-3"></a>         <span class="dt">token =</span> <span class="kw">Sys.getenv</span>(<span class="st">&quot;LDLINK_TOKEN&quot;</span>)</span>
<span id="cb14-4"><a href="#cb14-4"></a>        )</span></code></pre></div>
<pre><code>##     RS_number rs496202 rs201578600 rs11147477
## 1    rs496202    1.000       0.660      0.504
## 2 rs201578600    0.660       1.000      0.786
## 3  rs11147477    0.504       0.786      1.000</code></pre>
<p><br></p>
</div>
<div id="usage-multiple-query-variants-rsid-genomic-coordinates-multiple-populations-d" class="section level4">
<h4>Usage: Multiple query variants (rsID &amp; genomic coordinates), multiple populations, D’</h4>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1"></a><span class="kw">LDmatrix</span>(<span class="dt">snps =</span> <span class="kw">c</span>(<span class="st">&quot;chr13:32444611&quot;</span>, <span class="st">&quot;rs11147477&quot;</span>, <span class="st">&quot;rs201578600&quot;</span>), </span>
<span id="cb16-2"><a href="#cb16-2"></a>         <span class="dt">pop =</span> <span class="kw">c</span>(<span class="st">&quot;YRI&quot;</span>, <span class="st">&quot;CEU&quot;</span>), <span class="dt">r2d =</span> <span class="st">&quot;d&quot;</span>, </span>
<span id="cb16-3"><a href="#cb16-3"></a>         <span class="dt">token =</span> <span class="kw">Sys.getenv</span>(<span class="st">&quot;LDLINK_TOKEN&quot;</span>)</span>
<span id="cb16-4"><a href="#cb16-4"></a>        )</span></code></pre></div>
<pre><code>##     RS_number rs496202 rs201578600 rs11147477
## 1    rs496202    1.000       0.973      0.738
## 2 rs201578600    0.973       1.000      0.971
## 3  rs11147477    0.738       0.971      1.000</code></pre>
<p><br></p>
</div>
<div id="usage-multiple-query-variants-read-from-text-file-multiple-populations-d" class="section level4">
<h4>Usage: Multiple query variants read from text file, multiple populations, D’</h4>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1"></a>my_variants &lt;-<span class="st"> </span><span class="kw">read.table</span>(<span class="st">&quot;variant_list.txt&quot;</span>)</span>
<span id="cb18-2"><a href="#cb18-2"></a>my_variants</span></code></pre></div>
<pre><code>##           V1
## 1      rs456
## 2      rs114
## 3      rs127
## 4  rs7805287
## 5 rs60676332
## 6 rs10239961</code></pre>
<p>Then, call <em>LDmatrix</em> with:</p>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1"></a><span class="kw">LDmatrix</span>(<span class="dt">snps =</span> my_variants[,<span class="dv">1</span>], </span>
<span id="cb20-2"><a href="#cb20-2"></a>         <span class="dt">pop =</span> <span class="kw">c</span>(<span class="st">&quot;YRI&quot;</span>, <span class="st">&quot;CEU&quot;</span>), <span class="dt">r2d =</span> <span class="st">&quot;d&quot;</span>, </span>
<span id="cb20-3"><a href="#cb20-3"></a>         <span class="dt">token =</span> <span class="kw">Sys.getenv</span>(<span class="st">&quot;LDLINK_TOKEN&quot;</span>)</span>
<span id="cb20-4"><a href="#cb20-4"></a>        )</span></code></pre></div>
<pre><code>##    RS_number rs60676332 rs7805287 rs127 rs456 rs10239961 rs114
## 1 rs60676332      1.000     0.094 0.180 0.151      0.363 0.148
## 2  rs7805287      0.094     1.000 0.818 0.789      0.464 0.710
## 3      rs127      0.180     0.818 1.000 0.929      0.912 0.886
## 4      rs456      0.151     0.789 0.929 1.000      1.000 0.963
## 5 rs10239961      0.363     0.464 0.912 1.000      1.000 0.459
## 6      rs114      0.148     0.710 0.886 0.963      0.459 1.000</code></pre>
<p>Output is a table with rows and columns equal to the number of query variants and pairwise linkage disequilibrium statistics.</p>
<hr />
</div>
</div>
<div id="ldpair" class="section level3">
<h3>LDpair</h3>
<div id="function-2" class="section level4">
<h4>Function</h4>
<pre><code>LDpair(var1, var2, pop = &quot;CEU&quot;, token = NULL, output = &quot;table&quot;, file = FALSE)</code></pre>
<p>Investigates potentially correlated alleles for a pair of variants. Input is two query variants and a 1000 Genomes Project reference population(s) of interest.</p>
</div>
<div id="arguments-2" class="section level4">
<h4>Arguments</h4>
<ul>
<li><code>var1</code>, the first RS number (rsID) or genomic coordinate (GRCh37/hg19) (e.g. “chr7:24966446”), must match a bi-allelic variant</li>
<li><code>var2</code>, the second RS number or genomic coordinate, as above, must match a bi-allelic variant</li>
<li><code>pop</code>, a 1000 Genomes Project reference population, uses three letter population code, (e.g. YRI or CEU), multiple allowed, default = “CEU”</li>
<li><code>token</code>, LDlink provided user access token is required, default = NULL</li>
<li><code>output</code>, two output format options are available, “text”, which displays a two-by-two matrix displaying haplotype counts and allele frequencies along with other statistics, or “table”, which displays the same data in rows and columns, default = “table”</li>
<li><code>file</code>, optional character string naming a path and file for saving results. If file = FALSE, no file will be generated, default = FALSE</li>
</ul>
</div>
<div id="usage-with-output-argument-set-to-text" class="section level4">
<h4>Usage: With <code>output</code> argument set to “text”</h4>
<div class="sourceCode" id="cb23"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1"></a><span class="kw">LDpair</span>(<span class="dt">var1 =</span> <span class="st">&quot;rs496202&quot;</span>, </span>
<span id="cb23-2"><a href="#cb23-2"></a>       <span class="dt">var2 =</span> <span class="st">&quot;rs11147477&quot;</span>, </span>
<span id="cb23-3"><a href="#cb23-3"></a>       <span class="dt">pop =</span> <span class="st">&quot;YRI&quot;</span>, </span>
<span id="cb23-4"><a href="#cb23-4"></a>       <span class="dt">token =</span> <span class="kw">Sys.getenv</span>(<span class="st">&quot;LDLINK_TOKEN&quot;</span>), </span>
<span id="cb23-5"><a href="#cb23-5"></a>       <span class="dt">output =</span> <span class="st">&quot;text&quot;</span></span>
<span id="cb23-6"><a href="#cb23-6"></a>      )</span></code></pre></div>
<pre><code>## Query SNPs:
## rs496202 (chr13:32444611)
## rs11147477 (chr13:32509120)
## 
## YRI Haplotypes:
##                rs11147477
##                C       T
##              -----------------
##            C | 11    | 26    | 37    (0.171)
## rs496202     -----------------
##            G | 173   | 6     | 179   (0.829)
##              -----------------
##                184     32      216
##               (0.852) (0.148)
## 
##           G_C: 173 (0.801)
##           C_T: 26 (0.12)
##           C_C: 11 (0.051)
##           G_T: 6 (0.028)
## 
##           D&#39;: 0.7737
##           R2: 0.5037
##       Chi-sq: 108.8005
##      p-value: &lt;0.0001
## 
## rs496202(C) allele is correlated with rs11147477(T) allele
## rs496202(G) allele is correlated with rs11147477(C) allele</code></pre>
<p><br></p>
</div>
<div id="usage-with-no-output-argument-option-specified-using-default-table" class="section level4">
<h4>Usage: With no <code>output</code> argument option specified, using default “table”</h4>
<div class="sourceCode" id="cb25"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1"></a><span class="kw">LDpair</span>(<span class="dt">var1 =</span> <span class="st">&quot;rs496202&quot;</span>, </span>
<span id="cb25-2"><a href="#cb25-2"></a>       <span class="dt">var2 =</span> <span class="st">&quot;rs11147477&quot;</span>, </span>
<span id="cb25-3"><a href="#cb25-3"></a>       <span class="dt">pop =</span> <span class="st">&quot;YRI&quot;</span>, </span>
<span id="cb25-4"><a href="#cb25-4"></a>       <span class="dt">token =</span> <span class="kw">Sys.getenv</span>(<span class="st">&quot;LDLINK_TOKEN&quot;</span>)</span>
<span id="cb25-5"><a href="#cb25-5"></a>      )</span></code></pre></div>
<pre><code>##       var1       var2 pops       var1_pos       var2_pos var1_a1 var1_a2
## 1 rs496202 rs11147477  YRI chr13:32444611 chr13:32509120       C       G
##   var1_a1_freq var1_a2_freq var2_a1 var2_a2 var2_a1_freq var2_a2_freq
## 1        0.171        0.829       C       T        0.852        0.148
##   d_prime     r2    chisq p_val
## 1  0.7737 0.5037 108.8005 1e-04
##                                           corr_alleles
## 1 rs496202(C)-rs11147477(T), rs496202(G)-rs11147477(C)</code></pre>
<p>Output of the <code>output</code> argument “text” option is a two-by-two contingency table displaying haplotype counts and allele frequencies of the two query variants. Also displayed are calculated metrics of linkage disequilibrium including: D prime (D’), R square (R<sup>2</sup>), and goodness-of-fit (Chi-square and p-value). Goodness-of-fit tests for deviations of expected haplotype frequencies based on allele frequencies. Correlated alleles are reported if linkage disequilibrium is present (R<sup>2</sup> &gt; 0.1). If linkage equilibrium, no alleles are reported.</p>
<p>Output from the <code>output</code> argument “table” option converts the data from the two-by-two contingency table into a data frame.</p>
<hr />
</div>
</div>
<div id="ldpop" class="section level3">
<h3>LDpop</h3>
<div id="function-3" class="section level4">
<h4>Function</h4>
<pre><code>LDpop(var1, var2, pop = &quot;CEU&quot;, r2d = &quot;r2&quot;, token = NULL, file = FALSE)</code></pre>
<p>Investigates allele frequencies and linkage disequilibrium patterns across 1000G populations.</p>
</div>
<div id="arguments-3" class="section level4">
<h4>Arguments</h4>
<ul>
<li><code>var1</code>, the first RS number (rsID) or genomic coordinate (GRCh37/hg19) (e.g. “chr7:24966446”), must match a bi-allelic variant</li>
<li><code>var2</code>, the second RS number or genomic coordinate, as above, must match a bi-allelic variant</li>
<li><code>pop</code>, a 1000 Genomes Project reference population, uses three letter population code, (e.g. YRI or CEU), multiple allowed, default = “CEU”</li>
<li><code>r2d</code>, use “r2” if desired output is based on estimated R<sup>2</sup> or “d” if D’</li>
<li><code>token</code>, LDlink provided user access token is required, default = NULL</li>
<li><code>file</code>, optional character string naming a path and file for saving results. If file = FALSE, no file will be generated, default = FALSE</li>
</ul>
</div>
<div id="usage" class="section level4">
<h4>Usage</h4>
<div class="sourceCode" id="cb28"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb28-1"><a href="#cb28-1"></a><span class="kw">LDpop</span>(<span class="dt">var1 =</span> <span class="st">&quot;rs496202&quot;</span>, </span>
<span id="cb28-2"><a href="#cb28-2"></a>      <span class="dt">var2 =</span> <span class="st">&quot;rs11147477&quot;</span>, </span>
<span id="cb28-3"><a href="#cb28-3"></a>      <span class="dt">pop =</span> <span class="st">&quot;YRI&quot;</span>, </span>
<span id="cb28-4"><a href="#cb28-4"></a>      <span class="dt">r2d =</span> <span class="st">&quot;r2&quot;</span>, </span>
<span id="cb28-5"><a href="#cb28-5"></a>      <span class="dt">token =</span> <span class="kw">Sys.getenv</span>(<span class="st">&quot;LDLINK_TOKEN&quot;</span>)</span>
<span id="cb28-6"><a href="#cb28-6"></a>     )</span></code></pre></div>
<pre><code>##   Population   N rs496202_Allele_Freq rs11147477_Allele_Freq     R2     D&#39;
## 1        YRI 108 G: 82.87%, C: 17.13%   C: 85.19%, T: 14.81% 0.5037 0.7737</code></pre>
<hr />
</div>
</div>
<div id="ldproxy" class="section level3">
<h3>LDproxy</h3>
<div id="function-4" class="section level4">
<h4>Function</h4>
<pre><code>LDproxy(snp, pop = &quot;CEU&quot;, r2d = &quot;r2&quot;, token = NULL, file = FALSE)</code></pre>
<p>Explore proxy and putatively functional variants for a single query variant. Input is a single RS number and a population group. Depending on the number of query populations, this function could take some time to run.</p>
</div>
<div id="arguments-4" class="section level4">
<h4>Arguments</h4>
<ul>
<li><code>snp</code>, an RS number (rsID) or chromosome coordinate (GRCh37/hg19) (e.g. “chr7:24966446”), one per query, RS number must match a bi-allelic variant</li>
<li><code>pop</code>, a 1000 Genomes Project reference population, uses three letter population code, (e.g. YRI or CEU), multiple allowed, default = “CEU”</li>
<li><code>r2d</code>, use “r2” if desired output is based on estimated R<sup>2</sup> or “d” if D’</li>
<li><code>token</code>, LDlink provided user access token is required, default = NULL</li>
<li><code>file</code>, optional character string naming a path and file for saving results. If file = FALSE, no file will be generated, default = FALSE</li>
</ul>
</div>
<div id="usage-single-reference-population" class="section level4">
<h4>Usage: single reference population</h4>
<div class="sourceCode" id="cb31"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb31-1"><a href="#cb31-1"></a>my_proxies &lt;-<span class="st"> </span><span class="kw">LDproxy</span>(<span class="dt">snp =</span> <span class="st">&quot;rs456&quot;</span>, </span>
<span id="cb31-2"><a href="#cb31-2"></a>                      <span class="dt">pop =</span> <span class="st">&quot;YRI&quot;</span>, </span>
<span id="cb31-3"><a href="#cb31-3"></a>                      <span class="dt">r2d =</span> <span class="st">&quot;r2&quot;</span>, </span>
<span id="cb31-4"><a href="#cb31-4"></a>                      <span class="dt">token =</span> <span class="kw">Sys.getenv</span>(<span class="st">&quot;LDLINK_TOKEN&quot;</span>)</span>
<span id="cb31-5"><a href="#cb31-5"></a>                     )</span></code></pre></div>
<p>Output is a data frame stored in the variable <code>my_proxies</code> with 2455 rows and 10 columns with data.</p>
<div class="sourceCode" id="cb32"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb32-1"><a href="#cb32-1"></a><span class="kw">head</span>(my_proxies)</span></code></pre></div>
<pre><code>##    RS_Number         Coord Alleles    MAF Distance Dprime     R2
## 1      rs456 chr7:24962419   (G/C) 0.1944        0      1 1.0000
## 2      rs457 chr7:24962426   (T/C) 0.1944        7      1 1.0000
## 3 rs28475742 chr7:24964633   (G/T) 0.1944     2214      1 1.0000
## 4      rs123 chr7:24966446   (C/A) 0.1944     4027      1 1.0000
## 5      rs125 chr7:24959703   (C/T) 0.2037    -2716      1 0.9436
## 6      rs128 chr7:24958977   (C/T) 0.2037    -3442      1 0.9436
##   Correlated_Alleles RegulomeDB Function
## 1            G=G,C=C          5     &lt;NA&gt;
## 2            G=T,C=C          5     &lt;NA&gt;
## 3            G=G,C=T          4     &lt;NA&gt;
## 4            G=C,C=A         1f     &lt;NA&gt;
## 5            G=C,C=T          5     &lt;NA&gt;
## 6            G=C,C=T          7     &lt;NA&gt;</code></pre>
<p>Includes information on all variants -/+ 500 Kb of the query variant with a pairwise R<sup>2</sup> value greater than 0.01.</p>
<hr />
</div>
</div>
<div id="ldproxy_batch" class="section level3">
<h3>LDproxy_batch</h3>
<div id="function-5" class="section level4">
<h4>Function</h4>
<pre><code>LDproxy_batch(snp, pop = &quot;CEU&quot;, r2d = &quot;r2&quot;, token = NULL, append = FALSE)</code></pre>
<p>Query LDproxy using a list of query variants. <em>LDproxy_batch</em> will make sequential queries, one query per variant. Concurrent queries are not permitted by the LDlink API. Ouptput is saved as text file(s) to the current working directory. Depending on the number of query variants and reference populations selected, this function could time some time to run.</p>
</div>
<div id="arguments-5" class="section level4">
<h4>Arguments</h4>
<ul>
<li><code>snp</code>, a character string or data frame listing RS numbers (rsID) or chromosome coordinates (GRCh37/hg19) (e.g. “chr7:24966446”), one per line.</li>
<li><code>pop</code>, a 1000 Genomes Project reference population, uses three letter population code, (e.g. YRI or CEU), multiple allowed, default = “CEU”</li>
<li><code>r2d</code>, use “r2” if desired output is based on estimated R<sup>2</sup> or “d” if D’</li>
<li><code>token</code>, LDlink provided user access token is required, default = NULL</li>
<li><code>append</code>, a logical, if TRUE, output for eqch query variant is appended to a single text file and saved to the current working directory. If FALSE, output for each query variant is saved in its own text file with the query variant as the filename. Default value is FALSE.</li>
</ul>
</div>
<div id="usage-multiple-variants-default-pop-and-r2d" class="section level4">
<h4>Usage: multiple variants, default <code>pop</code> and <code>r2d</code></h4>
<p>The list of query variants passed to <em>LDproxy_batch</em> can be stored as a character string.</p>
<div class="sourceCode" id="cb35"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb35-1"><a href="#cb35-1"></a><span class="kw">LDproxy_batch</span>(<span class="dt">snp =</span> <span class="kw">c</span>(<span class="st">&quot;rs456&quot;</span>, <span class="st">&quot;rs114&quot;</span>, <span class="st">&quot;rs127&quot;</span>), </span>
<span id="cb35-2"><a href="#cb35-2"></a>              <span class="dt">token =</span> <span class="kw">Sys.getenv</span>(<span class="st">&quot;LDLINK_TOKEN&quot;</span>)</span>
<span id="cb35-3"><a href="#cb35-3"></a>             )</span></code></pre></div>
<p>Or, a longer list of variants can be read into a data frame from a text file and passed into <em>LDproxy_batch</em>. The list should be in a simple text file, one query variant per line. For example:</p>
<div class="sourceCode" id="cb36"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb36-1"><a href="#cb36-1"></a>my_variants &lt;-<span class="st"> </span><span class="kw">read.table</span>(<span class="st">&quot;variant_list.txt&quot;</span>)</span>
<span id="cb36-2"><a href="#cb36-2"></a>my_variants</span></code></pre></div>
<pre><code>##           V1
## 1      rs456
## 2      rs114
## 3      rs127
## 4  rs7805287
## 5 rs60676332
## 6 rs10239961</code></pre>
<p>Then, call <em>LDproxy_batch</em> with:</p>
<div class="sourceCode" id="cb38"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb38-1"><a href="#cb38-1"></a><span class="kw">LDproxy_batch</span>(<span class="dt">snp =</span> my_variants, </span>
<span id="cb38-2"><a href="#cb38-2"></a>              <span class="dt">token =</span> <span class="kw">Sys.getenv</span>(<span class="st">&quot;LDLINK_TOKEN&quot;</span>)</span>
<span id="cb38-3"><a href="#cb38-3"></a>             )</span></code></pre></div>
<p>Output not displayed. All output from <em>LDproxy_batch</em> is saved to a text file(s) in the current working directory.</p>
<hr />
</div>
</div>
<div id="snpchip" class="section level3">
<h3>SNPchip</h3>
<div id="function-6" class="section level4">
<h4>Function</h4>
<pre><code>SNPchip(snps, chip = &quot;ALL&quot;, token = NULL, file = FALSE)</code></pre>
<p>Used to find commercial genotyping chip arrays for variants. Input is a list of between 1 - 5000 variants (one per line) and desired commercial chip arrays to search. Input variants do not need to be on the same chromosome.</p>
</div>
<div id="arguments-6" class="section level4">
<h4>Arguments</h4>
<ul>
<li><code>snps</code>, between 1 - 5,000 variants, using an rsID or chromosome coordinate (e.g. “chr7:24966446”)</li>
<li><code>chip</code>, chip or arrays, platform code(s) for a SNP chip array, ALL_Illumina, ALL_Affy or ALL, default=ALL, use the <code>list_chips</code> utility (see below) to lookup available commercial SNP chip arrays and their codes.</li>
<li><code>token</code>, LDlink provided user access token is required is required, default = NULL</li>
<li><code>file</code>, optional character string naming a path and file for saving results. If file = FALSE, no file will be generated, default = FALSE</li>
</ul>
</div>
<div id="usage-multiple-variants-search-all-available-chip-arrays" class="section level4">
<h4>Usage: Multiple variants, search “ALL” available chip arrays</h4>
<div class="sourceCode" id="cb40"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb40-1"><a href="#cb40-1"></a><span class="kw">SNPchip</span>(<span class="dt">snps =</span> <span class="kw">c</span>(<span class="st">&quot;rs3&quot;</span>, <span class="st">&quot;rs4&quot;</span>, <span class="st">&quot;rs148890987&quot;</span>), </span>
<span id="cb40-2"><a href="#cb40-2"></a>        <span class="dt">chip =</span> <span class="st">&quot;ALL&quot;</span>, </span>
<span id="cb40-3"><a href="#cb40-3"></a>        <span class="dt">token =</span> <span class="kw">Sys.getenv</span>(<span class="st">&quot;LDLINK_TOKEN&quot;</span>)</span>
<span id="cb40-4"><a href="#cb40-4"></a>       )</span></code></pre></div>
<pre><code>## WARNING: The following RS number did not have any platforms found: rs148890987, rs3.</code></pre>
<pre><code>##     RS_Number Position_GRCh37 A_SNP5.0 A_CHB2 A_250S A_SNP6.0
## 1 rs148890987  chr13:32403784        0      0      0        0
## 2         rs3  chr13:32446842        0      0      0        0
## 3         rs4  chr13:32447222        1      1      1        1</code></pre>
<p><br></p>
</div>
<div id="usage-multiple-variants-search-two-affymetrix-arrays" class="section level4">
<h4>Usage: Multiple variants, search two Affymetrix arrays</h4>
<div class="sourceCode" id="cb43"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb43-1"><a href="#cb43-1"></a><span class="kw">SNPchip</span>(<span class="dt">snps =</span> <span class="kw">c</span>(<span class="st">&quot;rs3&quot;</span>, <span class="st">&quot;rs4&quot;</span>, <span class="st">&quot;rs148890987&quot;</span>), </span>
<span id="cb43-2"><a href="#cb43-2"></a>        <span class="dt">chip =</span> <span class="kw">c</span>(<span class="st">&quot;A_SNP5.0&quot;</span>, <span class="st">&quot;A_CHB2&quot;</span>), </span>
<span id="cb43-3"><a href="#cb43-3"></a>        <span class="dt">token =</span> <span class="kw">Sys.getenv</span>(<span class="st">&quot;LDLINK_TOKEN&quot;</span>)</span>
<span id="cb43-4"><a href="#cb43-4"></a>       )</span></code></pre></div>
<pre><code>## WARNING: The following RS number did not have any platforms found: rs148890987, rs3.</code></pre>
<pre><code>##     RS_Number Position_GRCh37 A_SNP5.0 A_CHB2
## 1 rs148890987  chr13:32403784        0      0
## 2         rs3  chr13:32446842        0      0
## 3         rs4  chr13:32447222        1      1</code></pre>
<p><br></p>
</div>
<div id="usage-multiple-variants-search-all-available-affymetrix-arrays-using-all_affy" class="section level4">
<h4>Usage: Multiple variants, search all available Affymetrix arrays using, “ALL_Affy”</h4>
<div class="sourceCode" id="cb46"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb46-1"><a href="#cb46-1"></a><span class="kw">SNPchip</span>(<span class="dt">snps =</span> <span class="kw">c</span>(<span class="st">&quot;rs3&quot;</span>, <span class="st">&quot;rs4&quot;</span>, <span class="st">&quot;rs148890987&quot;</span>), </span>
<span id="cb46-2"><a href="#cb46-2"></a>        <span class="dt">chip =</span> <span class="st">&quot;ALL_Affy&quot;</span>, </span>
<span id="cb46-3"><a href="#cb46-3"></a>        <span class="dt">token =</span> <span class="kw">Sys.getenv</span>(<span class="st">&quot;LDLINK_TOKEN&quot;</span>)</span>
<span id="cb46-4"><a href="#cb46-4"></a>       )</span></code></pre></div>
<pre><code>## WARNING: The following RS number did not have any platforms found: rs148890987, rs3.</code></pre>
<pre><code>##     RS_Number Position_GRCh37 A_SNP5.0 A_CHB2 A_250S A_SNP6.0
## 1 rs148890987  chr13:32403784        0      0      0        0
## 2         rs3  chr13:32446842        0      0      0        0
## 3         rs4  chr13:32447222        1      1      1        1</code></pre>
<p>Output is a data frame of query variant rows (RS number), genomic coordinate (GRCh37) and genotyping chip array columns. The presence of a “1” designates the variant is present on the respective commercial genotyping array and a “0” indicates that it is not present on the genotyping array.</p>
<hr />
</div>
</div>
<div id="snpclip" class="section level3">
<h3>SNPclip</h3>
<div id="function-7" class="section level4">
<h4>Function</h4>
<pre><code>SNPclip(snps, pop = &quot;CEU&quot;, r2_threshold = &quot;0.1&quot;, maf_threshold = &quot;0.01&quot;, token = NULL, file = FALSE)</code></pre>
<p>Prune a list of variants by linkage disequilibrium. Input is a list of variant RS numbers (one per line) and a population group.</p>
</div>
<div id="arguments-7" class="section level4">
<h4>Arguments</h4>
<ul>
<li><code>snps</code>, a list of between 1 - 5,000 variants, using an RS number (rsID) or chromosome coordinate (GRCh37) (e.g. “chr7:24966446”). All input variants must be on the same chromosome and match a bi-allelic variant.</li>
<li><code>pop</code>, a 1000 Genomes Project reference population, uses three letter population code, (e.g. YRI or CEU), multiple allowed, default = “CEU”</li>
<li><code>r2_threshold</code>, Used to set the R<sup>2</sup> threshold for LD pruning. One of each pair of variants with a R<sup>2</sup> greater than the threshold is removed. Value needs to be in the range 0 to 1. Default value is 0.1.</li>
<li><code>maf_threshold</code>, Used to set minor allele frequency (MAF) threshold for LD pruning. Variants with a MAF less than or equal to the threshold are removed. Value needs to be in the range 0 to 1. Default value is 0.01.</li>
<li><code>token</code>, LDlink provided user access token is required is required, default = NULL</li>
<li><code>file</code>, optional character string naming a path and file for saving results. If file = FALSE, no file will be generated, default = FALSE</li>
</ul>
</div>
<div id="usage-multiple-variants" class="section level4">
<h4>Usage: Multiple Variants</h4>
<div class="sourceCode" id="cb50"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb50-1"><a href="#cb50-1"></a><span class="kw">SNPclip</span>(<span class="dt">snps =</span>  <span class="kw">c</span>(<span class="st">&quot;rs3&quot;</span>, <span class="st">&quot;rs4&quot;</span>, <span class="st">&quot;rs148890987&quot;</span>, <span class="st">&quot;rs115955931&quot;</span>), </span>
<span id="cb50-2"><a href="#cb50-2"></a>        <span class="dt">pop =</span> <span class="st">&quot;YRI&quot;</span>, </span>
<span id="cb50-3"><a href="#cb50-3"></a>        <span class="dt">r2_threshold =</span>  <span class="st">&quot;0.1&quot;</span>, </span>
<span id="cb50-4"><a href="#cb50-4"></a>        <span class="dt">maf_threshold =</span> <span class="st">&quot;0.01&quot;</span>, </span>
<span id="cb50-5"><a href="#cb50-5"></a>        <span class="dt">token =</span> <span class="kw">Sys.getenv</span>(<span class="st">&quot;LDLINK_TOKEN&quot;</span>)</span>
<span id="cb50-6"><a href="#cb50-6"></a>       )</span></code></pre></div>
<pre><code>##     RS_Number       Position          Alleles
## 1         rs3 chr13:32446842 C=0.829, T=0.171
## 2         rs4 chr13:32447222 A=0.829, G=0.171
## 3 rs148890987 chr13:32403784     C=1.0, T=0.0
## 4 rs115955931 chr13:32130008 G=0.954, A=0.046
##                                             Details
## 1                                     Variant kept.
## 2 Variant in LD with rs3 (R2=1.0), variant removed.
## 3              Variant MAF is 0.0, variant removed.
## 4                                     Variant kept.</code></pre>
<p>The output table provides details including query varaint RS number, genomic position, alleles, and and details about whether the variant was kept or removed.</p>
<hr />
</div>
</div>
</div>
<div id="utilities" class="section level2">
<h2>Utilities</h2>
<div id="list_chips" class="section level3">
<h3>list_chips</h3>
<div id="function-8" class="section level4">
<h4>Function</h4>
<pre><code>list_chips()</code></pre>
<p>Provides a data frame listing the names and abbreviation codes for available commerical SNP Chip Arrays from Illumina and Affymetrix.</p>
</div>
<div id="usage-1" class="section level4">
<h4>Usage</h4>
<div class="sourceCode" id="cb53"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb53-1"><a href="#cb53-1"></a><span class="kw">list_chips</span>()</span></code></pre></div>
<pre><code>##         chip_code                               chip_name
## 1       A_Exome1A               Affymetrix Axiom Exome 1A
## 2      A_Exome319              Affymetrix Axiom Exome 319
## 3           A_AFR                 Affymetrix Axiom GW AFR
## 4           A_ASI                 Affymetrix Axiom GW ASI
## 5          A_CHB2                Affymetrix Axiom GW CHB2
## 6           A_EAS                 Affymetrix Axiom GW EAS
## 7           A_EUR                 Affymetrix Axiom GW EUR
## 8            A_Hu                  Affymetrix Axiom GW Hu
## 9        A_Hu-CHB              Affymetrix Axiom GW Hu-CHB
## 10          A_LAT                 Affymetrix Axiom GW LAT
## 11     A_DMETplus                    Affymetrix DMET Plus
## 12          A_10X           Affymetrix Mapping 10K Xba142
## 13         A_250N             Affymetrix Mapping 250K Nsp
## 14         A_250S             Affymetrix Mapping 250K Sty
## 15          A_50H          Affymetrix Mapping 50K Hind240
## 16          A_50X           Affymetrix Mapping 50K Xba240
## 17         A_Onco                     Affymetrix OncoScan
## 18      A_OncoCNV                 Affymetrix OncoScan CNV
## 19       A_SNP5.0                      Affymetrix SNP 5.0
## 20       A_SNP6.0                      Affymetrix SNP 6.0
## 21  I_CardioMetab              Illumina Cardio-MetaboChip
## 22         I_1M-D                  Illumina Human1M-Duov3
## 23           I_1M                      Illumina Human1Mv1
## 24        I_610-Q                Illumina Human610-Quadv1
## 25       I_660W-Q               Illumina Human660W-Quadv1
## 26       I_CNV-12                    Illumina HumanCNV-12
## 27     I_CNV370-D              Illumina HumanCNV370-Duov1
## 28     I_CNV370-Q             Illumina HumanCNV370-Quadv3
## 29          I_CVD                     Illumina HumanCVDv1
## 30      I_Core-12                 Illumina HumanCore-12v1
## 31   I_CoreE-12v1            Illumina HumanCoreExome-12v1
## 32 I_CoreE-12v1.1          Illumina HumanCoreExome-12v1.1
## 33   I_CoreE-24v1            Illumina HumanCoreExome-24v1
## 34 I_CoreE-24v1.1          Illumina HumanCoreExome-24v1.1
## 35    I_Cyto-12v2              Illumina HumanCytoSNP-12v2
## 36  I_Cyto-12v2.1            Illumina HumanCytoSNP-12v2.1
## 37 I_Cyto-12v2.1f       Illumina HumanCytoSNP-12v2.1 FFPE
## 38     I_Exome-12              Illumina HumanExome-12v1.1
## 39     I_Exon510S                Illumina HumanExon510Sv1
## 40         I_240S                   Illumina HumanHap240S
## 41        I_300-D              Illumina HumanHap300-Duov2
## 42          I_300                  Illumina HumanHap300v1
## 43        I_550v1                  Illumina HumanHap550v1
## 44        I_550v3                  Illumina HumanHap550v3
## 45         I_650Y                 Illumina HumanHap650Yv3
## 46  I_Immuno-24v1               Illumina HumanImmuno-24v1
## 47  I_Immuno-24v2               Illumina HumanImmuno-24v2
## 48   I_Linkage-12                Illumina HumanLinkage-12
## 49   I_Linkage-24                Illumina HumanLinkage-24
## 50        I_NS-12                     Illumina HumanNS-12
## 51         I_O1-Q              Illumina HumanOmni1-Quadv1
## 52        I_O1S-8                Illumina HumanOmni1S-8v1
## 53       I_O2.5-4               Illumina HumanOmni2.5-4v1
## 54       I_O2.5-8             Illumina HumanOmni2.5-8v1.2
## 55    I_O2.5E-8v1          Illumina HumanOmni2.5Exome-8v1
## 56  I_O2.5E-8v1.1        Illumina HumanOmni2.5Exome-8v1.1
## 57  I_O2.5E-8v1.2        Illumina HumanOmni2.5Exome-8v1.2
## 58      I_O2.5S-8              Illumina HumanOmni2.5S-8v1
## 59         I_O5-4                 Illumina HumanOmni5-4v1
## 60        I_O5E-4            Illumina HumanOmni5Exome-4v1
## 61        I_OE-12          Illumina HumanOmniExpress-12v1
## 62       I_OE-12f     Illumina HumanOmniExpress-12v1 FFPE
## 63        I_OE-24          Illumina HumanOmniExpress-24v1
## 64      I_OEE-8v1      Illumina HumanOmniExpressExome-8v1
## 65    I_OEE-8v1.1    Illumina HumanOmniExpressExome-8v1.1
## 66    I_OEE-8v1.2    Illumina HumanOmniExpressExome-8v1.2
## 67    I_OEE-8v1.3    Illumina HumanOmniExpressExome-8v1.3
## 68      I_OZH-8v1          Illumina HumanOmniZhongHua-8v1
## 69    I_OZH-8v1.1        Illumina HumanOmniZhongHua-8v1.1
## 70    I_OZH-8v1.2        Illumina HumanOmniZhongHua-8v1.2
## 71      I_Cyto850          Illumina Infinium CytoSNP-850K
## 72          I_100           Illumina Infinium Human100kv1
## 73  I_ME-Global-8 Illumina Infinium Multi-Ethnic Global-8
## 74    I_OncoArray        Illumina Infinium OncoArray-500K
## 75    I_Psyc-24v1       Illumina Infinium PsychArray-24v1
## 76  I_Psyc-24v1.1     Illumina Infinium PsychArray-24v1.1</code></pre>
<hr />
</div>
</div>
<div id="list_pop" class="section level3">
<h3>list_pop</h3>
<div id="function-9" class="section level4">
<h4>Function</h4>
<pre><code>list_pop()</code></pre>
<p>Provides a data frame listing the available reference populations from the 1000 Genomes Project, continetal or super-populations (e.g. European, African, Admixed American) and sub-populations (e.g Finnish, Gambian, Peruvian)</p>
</div>
<div id="usage-2" class="section level4">
<h4>Usage</h4>
<div class="sourceCode" id="cb56"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb56-1"><a href="#cb56-1"></a><span class="kw">list_pop</span>()</span></code></pre></div>
<pre><code>##    pop_code super_pop_code                                  pop_name
## 1       ALL            ALL                           ALL POPULATIONS
## 2       AFR            AFR                                   AFRICAN
## 3       YRI            AFR                  Yoruba in Ibadan, Nigera
## 4       LWK            AFR                    Luhya in Webuye, Kenya
## 5       GWD            AFR                 Gambian in Western Gambia
## 6       MSL            AFR                     Mende in Sierra Leone
## 7       ESN            AFR                            Esan in Nigera
## 8       ASW            AFR   Americans of African Ancestry in SW USA
## 9       ACB            AFR           African Carribbeans in Barbados
## 10      AMR            AMR                         AD MIXED AMERICAN
## 11      MXL            AMR    Mexican Ancestry from Los Angeles, USA
## 12      PUR            AMR            Puerto Ricans from Puerto Rico
## 13      CLM            AMR        Colombians from Medellin, Colombia
## 14      PEL            AMR                 Peruvians from Lima, Peru
## 15      EAS            EAS                                EAST ASIAN
## 16      CHB            EAS              Han Chinese in Bejing, China
## 17      JPT            EAS                  Japanese in Tokyo, Japan
## 18      CHS            EAS                      Southern Han Chinese
## 19      CDX            EAS       Chinese Dai in Xishuangbanna, China
## 20      KHV            EAS         Kinh in Ho Chi Minh City, Vietnam
## 21      EUR            EUR                                  EUROPEAN
## 22      CEU            EUR Utah Residents from North and West Europe
## 23      TSI            EUR                         Toscani in Italia
## 24      FIN            EUR                        Finnish in Finland
## 25      GBR            EUR           British in England and Scotland
## 26      IBS            EUR               Iberian population in Spain
## 27      SAS            SAS                               SOUTH ASIAN
## 28      GIH            SAS  Gujarati Indian from Houston, Texas, USA
## 29      PJL            SAS             Punjabi from Lahore, Pakistan
## 30      BEB            SAS                   Bengali from Bangladesh
## 31      STU            SAS              Sri Lankan Tamil from the UK
## 32      ITU            SAS                 Indian Telugu from the UK</code></pre>
<hr />
</div>
</div>
</div>
<div id="session-information" class="section level2">
<h2>Session Information</h2>
<div class="sourceCode" id="cb58"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb58-1"><a href="#cb58-1"></a><span class="kw">sessionInfo</span>()</span></code></pre></div>
<pre><code>## R version 3.5.2 (2018-12-20)
## Platform: x86_64-apple-darwin15.6.0 (64-bit)
## Running under: macOS Sierra 10.12.6
## 
## Matrix products: default
## BLAS: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRblas.0.dylib
## LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib
## 
## locale:
## [1] C/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8
## 
## attached base packages:
## [1] stats     graphics  grDevices utils     datasets  methods   base     
## 
## other attached packages:
## [1] LDlinkR_1.0.1
## 
## loaded via a namespace (and not attached):
##  [1] httr_1.4.1      compiler_3.5.2  R6_2.4.0        magrittr_1.5   
##  [5] htmltools_0.3.6 tools_3.5.2     yaml_2.2.0      Rcpp_1.0.0     
##  [9] stringi_1.2.4   rmarkdown_1.11  knitr_1.23      stringr_1.4.0  
## [13] digest_0.6.18   xfun_0.4        evaluate_0.13</code></pre>
</div>



<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
